uint32 num_links
geometry_msgs/Wrench[] wrenches
